docs/changelog.rst

```diff
@@ -0,0 +1,162 @@
+=========
+Changelog
+=========
+
+* Changes entry point commands from "_" to "-"
```

Comment:
I wonder if we want to display all of these items or just do this for the next releases?

Reply:
Good point. The Raiden guys only want to display "user-facing" releases. But that leads to a lot of `Add missing changelog entries`. Since we are deploying the MVP at the moment, I suggest, let's keep it that way for now and optimize later when we have actual users?  If not, we would need to filter useful PRs somehow.

Reply:
... ah reading your point again, you meant something different. Hmm I don't care tbh :) In case you have a preference, then I will change the changelog

Reply:
change it, as discussed